src/firefly/js/ui/TextButton.css

```diff
@@ -0,0 +1,20 @@
+
+.ff-MenuItem-linkButton {
+    padding: 5px;
+    -moz-border-radius: 0 0 5px 5px;
+    -webkit-border-radius: 0 0 5px 5px;
+    border-radius: 0 0 5px 5px;
+    background-color: rgba(245,245,245,.97);
+    -webkit-box-shadow: 0px 5px 10px rgba(0,0,0,0.3);
+    -moz-box-shadow: 0px 5px 10px rgba(0,0,0,0.3);
+    box-shadow: 0px 5px 10px rgba(0,0,0,0.3);
+}
+
+.ff-MenuItem-linkButton .menuItemText {
+    font-size: 10pt;
+    line-height: 14pt;
+}
+
+.ff-MenuItem-linkButton .menuItemText:HOVER {
+    background-color: rgba(255,255,255,.97);
+}
```

Comment:
I think, this css file has noting to do with the TextButton and needs to be deleted.